readyplayerme/Unity-WebGL-Example
# [3028451820] Update SDK + added git PR template

## [3028451820](https://ready-player-me.monday.com/boards/2563815861/pulses/3028451820)

## Description

-   Updated the Ready Player Me Unity SDK to v1.11.0




## Changes

#### Added

-   List your additions and new features here.
-   Render API example
-   SDK Logger
-   example loading multiple avatars



## How to Test

-   Run/use the example project as usual everything should work as before with some improvements added



## Checklist
